components/scream/src/share/util/scream_common_physics_functions.hpp

```diff
@@ -236,6 +260,20 @@ struct PhysicsFunctions
                              const Real surf_geopotential,
                              const view_1d<ScalarT>& dse);
 
+  template<typename ScalarT, typename InputProviderX, typename InputProviderQ>
+  KOKKOS_INLINE_FUNCTION
+  static void calculate_wetmmr_from_drymmr (const MemberType& team,
+                             const InputProviderX& drymmr,
+                             const InputProviderQ& qv,
+                             const view_1d<ScalarT>& wetmmr);
```

Comment:
I'm probably just being dense here, but why is the output wetmmr being declared const here?

Reply:
I believe that since we are passing by reference (&) that the reference is being declared as `const`, but the data in wetmmr can still be changed.

Reply:
Ok, that's what I suspected. Thanks.